umake/frameworks/baseinstaller.py

```diff
@@ -173,21 +174,24 @@ def get_metadata_and_check_license(self, result):
             logger.error("An error occurred while downloading {}: {}".format(self.download_page, error_msg))
             UI.return_main_screen()
 
-        url, checksum = (None, None)
-        with StringIO() as license_txt:
+        with StringIO() as license_text:
+            url, checksum = (None, None)
```

Comment:
Same with this, the change itself is fine, but has nothing to do with this PR. Please get that in logical commits and in a separate PR.